weave/index/index_sqlite.py

```diff
@@ -95,6 +95,9 @@ def generate_index(self, **kwargs):
 
         Parameters
         ----------
+        refresh: bool (default: False)
```

Comment:
Was this supposed to be added to the generate_index docstring?

Reply:
Good catch--removed 👍